Create a user testing session template

christinach opened the pull request.

christinach committed:
Create a user testing session template

kevinreiss reviewed (approved):
Thanks for creating this, these guidelines and suggestions are good starting place for planning the FABLE sessions in the future.

Ka-YT commented:
Recommended practices for Fable sessions:
[Google Doc](https://docs.google.com/document/d/10zNwh6DTeAwEAETP7kBLGAuJoKlHQCBQqeJ6Hq2UhDw/edit?usp=sharing)

christinach committed:
Update the template following
https://docs.google.com/document/d/10zNwh6DTeAwEAETP7kBLGAuJoKlHQCBQqeJ6Hq2UhDw/edit?tab=t.0 from @Ka-YT

christinach committed:
Add link to example script
